pkg/util/iptables/iptables.go

```diff
@@ -120,6 +120,56 @@ const MinCheckVersion = "1.4.11"
 const MinWaitVersion = "1.4.20"
 const MinWait2Version = "1.4.22"
 
+// no-op implemenatation of Interface
+type fake struct{}
+
+func NewFake() *fake {
```

Comment:
It'll probably be useful for integration tests as well. AFAIU we don't have proxy in current integration tests. I'll move it to the fake_* file. I don't really know how to solve the linking problem nicely:
- IIUC moving fake to a separate file helps with readability, but it's still linked into real binaries
- moving it to a subdirectory requires one of two things:
  - redeclaring interface in the 'test' directory
  - including both 'real' implementation and 'test' one in every place where 'test' one is used.

Both options are bad IMO...